courseware/data_literacy/index.md

```diff
@@ -4,32 +4,28 @@ title: Data literacy with Data Commons
 nav_order: 221
 parent: Educational materials
 grand_parent: Additional resources
-redirect_from: /courseware/data_literacy/overview
+has_children: true
+redirect_from: 
+    - /courseware/data_literacy/overview
+    - /courseware/data_literacy/faq
 ---
 
+{:.no_toc}
 # Data literacy with Data Commons
 
 <div markdown="span" class="alert alert-info" role="alert" style="text-align: center">
-    >> [Access the course materials here](course_materials/). <<
+    >> [Access the course materials here](how_to_use.md). <<
```

Comment:
What do you think of linking to the modules page here instead? Otherwise, users have to click through the how to use page before getting to the modules.

Reply:
Makes sense.